mne/decoding/ems.py

```diff
@@ -67,7 +71,7 @@ def fit(self, X, y):
         """
         classes = np.unique(y)
         if len(classes) != 2:
-            raise ValueError('EMS only works for binary classification.')
+            raise ValueError('EMS only works for binary classification. %s' % y)  # noqa: E501
```

Comment:
```suggestion
            raise ValueError(
                'EMS only works for binary classification. %s' % y
            )
```

let's avoid the # noqa: E501 when we can